components/article/client/view.coffee

```diff
@@ -56,12 +56,14 @@ module.exports = class ArticleView extends Backbone.View
     @renderCalloutSections()
     @setupFooterArticles()
     @setupStickyShare()
+    @setupMobileShare(@article)
```

Comment:
Since we already have `@article` available in the view, we shouldn't have to pass it in the args